specificatie/openapi.yaml

```diff
@@ -63,7 +63,25 @@ paths:
             type: integer
             minimum: 1
             maximum: 99999
-        - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/parameters/fields"
+        - in: query
+          name: huisnummertoevoeging
+          description: Huisnummertoevoeging behorende bij een nummeraanduiding.
+          required: false
+          schema:
+            type: string
+            pattern: "^[0-9a-zA-Z]{1,4}$"
+            example: "bis"
+        - in: query
+          name: huisletter
+          description: Huisletter behorende bij een nummeraanduiding.
+          required: false
+          schema:
+            type: string
+            pattern: "^[a-zA-Z]{1}$"
```

Comment:
Moet hier i.p.v. '{1}' in het pattern geen gebruik gemaakt worden van minLength en maxLength?
Zie VNG-Realisatie/Haal-Centraal-common#112

Reply:
Ook hier n.a.v. de discussie van vandaag maxLength en minLength verwidjerd en het pattern weer aangepast.